graviti/portex/field.py

```diff
@@ -11,11 +11,95 @@
 
 from graviti.portex.base import _INDENT, PortexType
 from graviti.portex.package import Imports
-from graviti.utility import NameOrderedDict
+from graviti.utility import InmutableNameOrderedDict, NameOrderedDict
 
 _T = TypeVar("_T", bound="Fields")
 
 
+class InmutableFields(InmutableNameOrderedDict[PortexType]):
+    """Represents an inmutable fields dict."""
+
+    def __repr__(self) -> str:
+        return self._repr1(0)
+
+    @classmethod
+    def _check_value(cls, value: PortexType) -> None:
+        if not isinstance(value, PortexType):
+            raise TypeError(f'The value in "{cls.__name__}" should be a PortexType')
+
+    def _repr1(self, level: int) -> str:
```

Comment:
what is the difference between the `__repr__` in `InmutableFields` and normal `Fields`?

Reply:
No difference.

Reply:
`Fields` will be replaced with `ConnectedFields` in the future, do we need to reuse this part of logic now?